mereswine/crawler.py

```diff
@@ -44,9 +32,31 @@ def crawl(instance):
 
 
 def geolocate(instance):
-    url = 'http://freegeoip.net/json/{0}'.format(trim_url(instance.url))
-    r = requests.get(url)
+    api_key = current_app.config.get('IP_GEOLOCATION_API_KEY')
```

Comment:
If we load the example config to get default values you can use `[]` here since it'll always be defined